### PR TITLE
Fabo/simpler hide tx meta

### PR DESCRIPTION
**Pull request by faboweb**
Simpler version of #3583 

Closes #3583

### CODE REVIEW

File: src/components/transactions/TransactionItem.vue [hunk 1]
```diff
@@ -106,10 +106,9 @@ export default {
     }
   },
   methods: {
-    toggleDetail(event) {
-      if (event.target.className !== `address`) {
```
Comment (faboweb) [inline src/components/transactions/TransactionItem.vue]: @Bitcoinera this was not a problem anymore?

File: src/components/transactions/message-view/WithdrawDelegationRewardMessageDetails.vue [hunk 2]
```diff
@@ -83,10 +81,10 @@
...
 
 <script>
-import { atoms, viewDenom, prettyLong } from "scripts/num.js"
-import { resolveValidatorName } from "src/filters"
-import TransactionIcon from "../TransactionIcon"
-import Avatar from "common/Avatar"
+import { atoms, viewDenom, prettyLong } from 'scripts/num.js'
+import { resolveValidatorName } from 'src/filters'
+import TransactionIcon from '../TransactionIcon'
+import Avatar from 'common/Avatar'
```
Comment (Bitcoinera) [inline src/components/transactions/message-view/WithdrawDelegationRewardMessageDetails.vue]: Why is the linter changing now all this?

**Comment by Bitcoinera** [comment]
I'd like to merge #3591 here

File: src/components/transactions/message-view/WithdrawDelegationRewardMessageDetails.vue (hunk 2 above)
Comment (faboweb) [reply src/components/transactions/message-view/WithdrawDelegationRewardMessageDetails.vue]: good question o.O

**Comment by faboweb** [comment]
> I'd like to merge #3591 here

refused. let'S create small PRs and not blow up small ones ;P

**Comment by Bitcoinera** [comment]
> refused. let'S create small PRs and not blow up small ones ;P

😿

**Pull request merged**